kubernetes/FAQs.md

```diff
@@ -0,0 +1,84 @@
+## Introduction
+
+From Kubernetes v1.13, CoreDNS is the recommended default DNS server in Kubernetes. 
+
+As more and more users are deploying and using CoreDNS is their Kubernetes cluster, there are a number of FAQs asked by users having trouble setting up CoreDNS, or maintaining CoreDNS. 
+This document aims to maintain a collection of the most frequently asked questions by users during the support provided to users having trouble with the DNS Service after installation or upgrade of their Kubernetes cluster.
+
+## Useful links about CoreDNS for Kubernetes:
+
+- [Using CoreDNS for Service Discovery](https://kubernetes.io/docs/tasks/administer-cluster/coredns/)
+- [Debugging DNS Resolution](https://kubernetes.io/docs/tasks/administer-cluster/dns-debugging-resolution/)
+- [Troubleshooting kubeadm](https://kubernetes.io/docs/setup/independent/troubleshooting-kubeadm/)
+- [DNS Specifications for Kubernetes](https://github.com/kubernetes/dns/blob/master/docs/specification.md)
+- [CoreDNS Kubernetes plugin documentation](https://coredns.io/plugins/kubernetes/)
+
+## Pre-requisites before troubleshooting
+
+It is important to check whether the Kubernetes Network is stable before starting to troubleshoot DNS. It may appear like DNS isn't functioning when there is a network issue in the Kubernetes cluster. 
+In other words, an issue like "DNS does not work properly" has often its root cause as "Network is not working properly".
+
+Hence, if you're confident that the network is stable and DNS still doesn't work, this FAQ document is applicable. 
+
+## Known Issues
+
+### Some Prometheus data is missing after reload
+
+In CoreDNS v1.2.2 that is provided with K8s v1.12, after editing the ConfigMap,
+CoreDNS will automatically "reload" its configuration file.
+
+The reload causes some metrics on the Prometheus interface to be missing.
```

Comment:
I tried to follow open/closed issues and it seems like this had been fixed at some point.
@fturib can you confirm this?

Reply:
This is the only thing we list as a known issue?
Should we list all bugs that were in k8s released CoreDNS versions?
Or instead, just stick to frequently asked questions.

Reply:
I feel it is better to remove this section. 
If we have enough 'solved and unsolved' know issues, we can make it a separate doc, @fturib?

Reply:
> Should we list all bugs that were in k8s released CoreDNS versions?

IMO, this would take a very long time to compile.  We should probably remove this section.

If people have frequently asked "why are metrics not working"... and this was the reason, then we can phrase this as FAQ... but I dont recall it's something that got asked about...

Reply:
ok to remove.